lms/djangoapps/instructor/utils.py

```diff
@@ -186,7 +187,13 @@ def collect_ora2_data(course_id, include_email=False):
     cursor.execute(raw_queries[0])
     cursor.execute(raw_queries[1], [course_id])
     header = [item[0] for item in cursor.description]
-    return header, cursor.fetchall()
+    header.append('formatted_answer')
+    raw_answer_index = 4
+    data = cursor.fetchall()
+    data_rows = [list(row) + [dumps(
+        loads(row[raw_answer_index]),
+        ensure_ascii=False)] for row in data]
```

Comment:
This line does the following:

for each row, grab the raw_answer (which is a JSON string), load it as a JSON object (a Python map), and then re-serialize it as a JSON object (without enforcing ASCII constraints to ensure that the Unicode characters are not escaped)